src/Main.hs

```diff
@@ -51,7 +53,7 @@ ppException a = catch a ehandler
     ehandler e = maybe (throw e) (*> exitFailure)
                  (pSqlError <$> fromException e)
     bsToString = T.unpack . T.decodeUtf8
-    pSqlError e = mapM_ putStrLn
+    pSqlError e = mapM_ (hPutStrLn stderr)
```

Comment:
This is a visible change in behavior, I'm hesitant on this. As far as changes go, this is one of the more benign ones, but does change the behavior

```
postgresql-simple-migration --help | grep foo
postgresql-simple-migration --help | less
postgresql-simple-migration --help 2>&1 | grep foo
```

Reply:
1. None of the commands you provided as an example are related to this line in an any way
2. There’s no such argument `--help`, I added it to #35 but didn’t in this MR, the command will fail
3. This will print to `stderr` only in case there were some errors while applying migrations

If you refer to the usage of `printUsage`:
1. If you provide correct `-h` argument it will, as before, print usage info to `stdout` thus, if you use the program correctly, it will work as before.
2. If you provide **incorrect** arguments the program, as before, will fail and (this changed in this MR) will print usage info to `stderr` as it should. If you rely somewhere on failure scenario caused by using incorrect arguments then you’re doing something wrong and you’re using the program incorrectly. I think you should not pay any attention to copying bugs in order to keep behavior the same unless you build some platform emulation tool (e.g. WineHQ).
3. Someone may try to use `--help` argument (which doesn’t exist), it’s the most default one for getting usage info. But to whom I’m telling this, you just tried to do so in your example! And in Bash, if you don’t use `set -eo pipefail`, the grepping from your example would succeed (before this change) but this is so only because Bash sucks, it sweeps errors under the carpet. It is still a **bug**. But just for this case I thought it was a good idea to add `--help` argument alongwith `-h` one as a part of the MR (which I did in #35 but you were against it, I still think it is a good idea).